cmake_modules/parse_hpcc_chart.sh

```diff
@@ -0,0 +1,79 @@
+#!/bin/bash
+#
+# Common utilities used by tools for automating tagging and release
+#
+set -e
+
+function parse_chart()
+{
+  local _chart=$1
+  HPCC_PROJECT=$(grep -m1 '^name:' $_chart | sed "s/^name: *//")
+  HPCC_NAME="Helm Chart for $HPCC_PROJECT"
+  HPCC_VERSION=$(grep -m1 '^version:' $_chart | sed "s/^version: *//")
+
+  HPCC_MAJOR=$(echo $HPCC_VERSION | awk 'BEGIN {FS="[.-]"}; {print $1};')
+  HPCC_MINOR=$(echo $HPCC_VERSION | awk 'BEGIN {FS="[.-]"}; {print $2};')
+  HPCC_POINT=$(echo $HPCC_VERSION | awk 'BEGIN {FS="[.-]"}; {print $3};')
+  HPCC_MATURITY=$(echo $HPCC_VERSION | awk 'BEGIN {FS="[-.+]"}; {print $4};' | sed 's/[0-9]//g')
+  HPCC_SEQUENCE=$(echo $HPCC_VERSION | awk 'BEGIN {FS="[-.+]"}; {print $4};' | sed 's/[^0-9]//g')
+
+  # translate maturity/sequence to what go_rc/go_gold understand
+  if [ -z "$HPCC_MATURITY" ] ; then
+    HPCC_MATURITY=release
+  fi
+  # translate maturity to what go_rc/go_gold understand
+  if [ -z "$HPCC_SEQUENCE" ] ; then
+    HPCC_SEQUENCE=1
+  fi
+
+    echo "file $VERSIONFILE"
```

Comment:
indentation, and do you want to keep these lines in?  not currently called, so probably doesn't matter.